Source/WebCore/editing/ApplyStyleCommand.cpp

```diff
@@ -988,7 +992,7 @@ RefPtr<HTMLElement> ApplyStyleCommand::highestAncestorWithConflictingInlineStyle
 
     for (RefPtr ancestor = node; ancestor; ancestor = ancestor->parentNode()) {
         if (is<HTMLElement>(*ancestor) && shouldRemoveInlineStyleFromElement(style, downcast<HTMLElement>(*ancestor)))
```

Comment:
Maybe this should be written in terms of `dynamicDowncast`, instead of checking `is<HTMLElement>` then down casting multiple times?